File: atroposServer/getNextNovelNode.py
#!/usr/bin/python
import cgi
import json
import pygold

print "Content-type: text/json\n\n"

queryParams = cgi.FieldStorage()

if queryParams.has_key("choice") and queryParams["choice"].value !="":
	if (queryParams["choice"].value == "0"):
		data = {"$type": "DialogueNode, Assembly-CSharp", "dialogue":"You ain't half bad yourself!"}
	else:			
		data = {"$type": "DialogueNode, Assembly-CSharp", "dialogue":"Geez, you're makin me blush."}
	print json.dumps(data)
else:
	data = {"$type": "MenuNode, Assembly-CSharp", "choices":[ "You are cool", "You are suuuper cool" ]}
	print json.dumps(data)

File: atroposServer/getNextNovelNode.php
<?php

  /* create one master array of the records */
  $node = array();

  /* check for choice */
  if(isset($_GET['choice'])) {
    
    $node["__type"] = "DialogueNode";

    switch($_GET['choice']) {
	case 0:
	    $node["dialogue"] = "You ain't half bad yourself!";
	    break;
        case 1:
            $node["dialogue"] = "Geez, you're makin me blush.";
            break;
    }
  }
  else
  {
    $node["__type"] = "MenuNode";
    $node["choices"] = array("You are cool", "You are suuuper cool");
  }

  /* output in necessary format */
  header('Content-type: application/json');
  echo json_encode(array($node));

?>